packages/atomic/cypress/integration/result-list/folded-result-list-utils.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  generateComponentHTML,
+  ResultWithFolding,
+  SearchFoldedResponseSuccess,
+} from '../../fixtures/test-fixture';
+
+export function withFoldedResult(
```

Comment:
this could be moved into `folded-result-list-actions.ts` along with `addFoldedResultList`

Reply:
I forgot to remove this, I'm not using it anymore!